### ISSUE TITLE
Add selected Namespace/Assembly/Type to filters list

### ISSUE THREAD

**Issue by Malivil** [axodox/AxoTools#88]
It would be nice to be able to right-click on a node in the Report tree and click "Exclude" to automatically add it to the filters list. Right now I can do it manually but sometimes I have trouble figuring out what the filter string is supposed to be.

**Comment by axodox** [comment]
Good idea, I don't know when I will have time, but this should be quick to do.

**Comment by poolofthought** [comment]
I don't know if you can vote up options, but holy goodness I like this idea!

**Comment by nedmech** [comment]
Old issue, but I'd like to chime in on the desirability of this feature. It's a really awesome idea!
